<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d2_ProductItem</class>
 <widget class="QPushButton" name="d2_ProductItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>586</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#d2_ProductItem {
    background-color: rgb(255, 255, 255);
    border-radius:  12px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="backgroundWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#backgroundWidget {
    background-color: rgb(255, 255, 255);
    border-radius:  15px;
}
</string>
     </property>
     <widget class="QLabel" name="d2_lblName">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>478</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Baloo Cyrillic</family>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="d2_lblStatus">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="d2_lblPrice">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>530</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Baloo Cyrillic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 117, 17);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="d2_lblNumberBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Baloo Cyrillic</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#d2_lblNumberBackground{
	color: rgb(255, 255, 255);
	border-image: url(:/desing2/background/number-background.png);
	border-top-left-radius: 12px;
}

#d2_lblNumberBackground::disabled{
	color: rgb(255, 255, 255);
	border-image: url(:/desing2/background/number-background - disable.png);
	border-top-left-radius: 12px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="margin">
       <number>25</number>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="d2_lblNumber">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>10</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="d2_widgetImage" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>391</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="d2_lblImage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	border-image: url(:/desing2/1080x1920/png/toy/Slime_ninja.png)0 0 0 0 stretch stretch;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>d2_widgetImage</zorder>
     <zorder>d2_lblName</zorder>
     <zorder>d2_lblStatus</zorder>
     <zorder>d2_lblPrice</zorder>
     <zorder>d2_lblNumberBackground</zorder>
     <zorder>d2_lblNumber</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
